src/Context/BasicContext.php

```diff
@@ -442,6 +442,24 @@ public function stepIPressTheButton($text)
         $button->click();
     }
 
+    /**
+     * @Given /^I press the "([^"]*)" buttons$/
```

Comment:
Last minor thing - maybe you could make the `s` in buttons optional

Reply:
If I do that, the rule could be confused with the single button rule.

Reply:
Lol, ok